utilities/check_boundaries.py

```diff
@@ -102,10 +102,19 @@ def compare_features(idA, idB):
       return 3
   return 1
 
-def conv_date(datestr):
+def conv_date(datestr,is_start):
   if datestr == 'present':
     return '2100:01:01'
-  return datestr
+  args = datestr.split(':')
```

Comment:
This script wasn't updated to distinguish between year-only or year-and-month-only datestr designations.